runtime/print.cc

```diff
@@ -289,11 +289,7 @@ class printer : public expr_visitor, public stmt_visitor {
 
   void visit(const loop* l) override {
     *this << indent() << l->sym << " = loop(";
-    switch (l->max_workers) {
```

Comment:
Bring this back? printed IR will be pretty noisy with weird constants without this